frontend/src/components/profile/components/profile-info/profile-info.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { IUser } from 'common/interfaces/user';
+import { DateFormat } from 'common/enums';
+import { getFormattedDate } from 'helpers';
+import clsx from "clsx";
+import styles from './styles.module.scss';
+
+type Props = {
+  user: IUser
+  edit: () => void
```

Comment:
Create a type `type BindingCb = () => void` and put it to the shared folder(`src/common/types/cbs`)